modules/aruco/test/test_precomp.hpp

```diff
@@ -7,7 +7,6 @@
 #include "opencv2/ts.hpp"
 #include "opencv2/imgproc.hpp"
 #include "opencv2/calib3d.hpp"
-#include "opencv2/aruco.hpp"
 #include <opencv2/aruco/charuco.hpp>
```

Comment:
What is the purpose of this change?

If you want to remove module's headers, then remove them all (charuco.hpp is still here).

Currently it looks like that something was broken and there is a hack to hide that (so user code may be broken too).